frontend/server/src/Controllers/Problem.php

```diff
@@ -4742,6 +4743,30 @@ public static function getProblemListForSmarty(
         ];
     }
 
+    /**
+     *
+     * @return array{smartyProperties: array{payload: ProblemListCollection, title: string}, entrypoint: string}
+     *
```

Comment:
```suggestion
     * @return array{smartyProperties: array{payload: ProblemListCollection, title: string}, entrypoint: string}
     *
     * @omegaup-request-param string $problemLevelTag
```

para que no se queje psalm.

Reply:
esto no hay que actualizarlo a usar `title: \OmegaUp\TranslationString`?

```suggestion
     *
     * @return array{smartyProperties: array{payload: ProblemListCollection, title: \OmegaUp\TranslationString}, entrypoint: string}
     *
```